optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/spot/Spot.java

```diff
@@ -31,28 +33,28 @@
 
 @Entity
 @NamedQueries({
-        @NamedQuery(name = "Spot.findAll",
-                query = "select distinct s from Spot s left join fetch s.requiredSkill" +
-                        " where s.tenantId = :tenantId" +
-                        " order by s.name"),
+               @NamedQuery(name = "Spot.findAll",
+                           query = "select distinct s from Spot s left join fetch s.requiredSkills" +
+                                   " where s.tenantId = :tenantId" +
+                                   " order by s.name"),
 })
 @Table(uniqueConstraints = {@UniqueConstraint(columnNames = {"tenantId", "name"}), @UniqueConstraint(columnNames = {"id"})})
 public class Spot extends AbstractPersistable {
 
-    @NotNull @Size(min = 1, max = 120)
+    @NotNull
+    @Size(min = 1, max = 120)
     private String name;
     @NotNull
-    @ManyToOne
-    private Skill requiredSkill;
+    @ManyToMany(fetch = FetchType.EAGER)
+    private List<Skill> requiredSkills;
```

Comment:
Change "requiredSkills" to "requiredSkillList" for consistency with the rest of the domain code. Also apply that name change through the rest of this PR (including screens etc).
The philosophy is that we use plural for arrays (items), List suffix for lists (itemList), Set suffix for sets (itemSet), and so forth (itemMap), ...